app/src/main/java/org/stepic/droid/core/presenters/PersistentCourseListPresenter.kt

```diff
@@ -52,12 +58,14 @@ constructor(
         private const val MAX_CURRENT_NUMBER_OF_TASKS = 2
         private const val SEVEN_DAYS_MILLIS = 7 * 24 * 60 * 60 * 1000L
         private const val MILLIS_IN_SECOND = 1000L
+        private const val MY_COURSES_LOADING = "my_courses_loading" // Trace key
```

Comment:
Maybe we should store that constants like ones for analytics or at least add some prefix